src/emmental/learner.py

```diff
@@ -535,22 +535,32 @@ def _aggregate_running_metrics(
 
     def _set_learning_counter(self) -> None:
         if Meta.config["learner_config"]["n_steps"]:
-            self.start_epoch = 0
-            self.end_epoch = 1
             if Meta.config["learner_config"]["skip_learned_data"]:
+                self.start_epoch = 0
```

Comment:
So what you've done is cleaner. I didn't add epochs in the beginning because the data loader will reset the data order the same for each epoch (I'm fairly certain) so it doesn't really matter what epoch you resume from, you'll see the same order of data. But this is cleaner.

Reply:
Yes, you are right. I just want to make it consistent and clear.